addons/l10n_es/data/account_data.xml

```diff
@@ -6,13 +6,13 @@
         <record id="tax_group_iva_0" model="account.tax.group">
             <field name="name">IVA 0%</field>
         </record>
-        <record id="tax_group_iva_05" model="account.tax.group">
+        <record id="tax_group_iva_0-5" model="account.tax.group">
             <field name="name">IVA 0.5%</field>
```

Comment:
Some of these taxes are incorrectly tagged as `IVA`. This is for example `Recargo de equivalencia`, and others (like 19% is `Retenciones`)

Reply:
Ok, I'll change tax_group_iva_0-5's name to Recargo de equivalencia, and correct the name for 19% as well. Are there others ? If so, could you give me a complete list of the changes to do ?

Reply:
You can easily deduce them by the tax name they have. Example:

https://github.com/odoo/odoo/blob/65f5fc6bf8442fc7004d0fc8e006184359f0fe4e/addons/l10n_es/data/account_tax_data.xml#L586

or 

https://github.com/odoo/odoo/blob/65f5fc6bf8442fc7004d0fc8e006184359f0fe4e/addons/l10n_es/data/account_tax_data.xml#L250